go.sum

```diff
@@ -1,5 +1,8 @@
+fyne.io/fyne v1.4.3 h1:356CnXCiYrrfaLGsB7qLK3c6ktzyh8WR05v/2RBu51I=
+fyne.io/fyne v1.4.3/go.mod h1:8kiPBNSDmuplxs9WnKCkaWYqbcXFy0DeAzwa6PBO9Z8=
 github.com/Kodeworks/golang-image-ico v0.0.0-20141118225523-73f0f4cfade9 h1:1ltqoej5GtaWF8jaiA49HwsZD459jqm9YFz9ZtMFpQA=
 github.com/Kodeworks/golang-image-ico v0.0.0-20141118225523-73f0f4cfade9/go.mod h1:7uhhqiBaR4CpN0k9rMjOtjpcfGd6DG2m04zQxKnWQ0I=
+github.com/adrianre12/fyne v1.4.2 h1:Zoiqls4+3e3Vz5gEK6wJZh4cMQDkZ44cMd6gGVlvOx0=
```

Comment:
This also seems a little strange. Why is this included in the modules?

Reply:
Good catch, havn't a clue why it is in there. Go tidy sorted it out.